terraform/aws/development-infrastructure/synthetics-canary/README.md

```diff
@@ -25,3 +25,7 @@ Make URLs a list instead of a single value
 Consider using email address list outside of cloudwatch synthetics for
 other SNS topics
 
+
+# apply manually outside of pipeline
+terraform plan -var-file=inputs.tfvars -var 'synthetics_canary_create=true'
```

Comment:
missing an '=' after var

Reply:
for some reason terraform seems inconsistent with their flags, see https://developer.hashicorp.com/terraform/cli/commands/plan